scripts/run-e2e.ts

```diff
@@ -48,12 +48,11 @@ const clearPorts = (): void => {
   }
   for (const port of [ports.intake, ports.ehr, ports.backend]) {
     try {
-      const pid = execSync(`lsof -ti :${port}`).toString().trim();
-      if (pid) {
-        process.kill(parseInt(pid, 10), 'SIGTERM');
-      }
+      const output = execSync(`lsof -i :${port} | grep "^node" | awk '{print $2}'`).toString().trim();
```

Comment:
I ran `npm run apps:start`, after which I ran the tests and the console displayed `No process found on port 3000`. Did you check this case?